utils/gyb.py

```diff
@@ -716,7 +716,8 @@ def execute(self, context):
 
         # If we got a result, the code was an expression, so append
         # its value
-        if result is not None and result != '':
+        if result is not None \
+                or (isinstance(result, basestring) and result != ''):
```

Comment:
This won't work in Python 3. I think we try to make all the python code compatible with both.